.github/workflows/pipeline.yml

```diff
@@ -20,23 +20,23 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - label: Python 3.6 / SQLite / Monolith
+          - label: Py3.6 / SQLite / Monolith
```

Comment:
bionic has SQLite 3.22.0 - it can be useful to list it, we've had issues with older SQLite's in the past.